Kitodo/src/main/java/org/kitodo/export/ExportDms.java

```diff
@@ -86,6 +89,17 @@ public void startExport(Process process) throws DataException {
         }
     }
 
+    private boolean allChildrenExported(Process process) {
+        if (process.getChildren().isEmpty()) {
```

Comment:
```suggestion
        if (!process.getChildren().isEmpty()) {
```
Shouldn't this be negated?